Include/pymacro.h

```diff
@@ -10,6 +10,14 @@
 #  define static_assert _Static_assert
 #endif
 
+// static_assert is defined in GLIB from version 2.16. Before it requires
```

Comment:
GLIBC? (GLIB [exists](https://en.wikipedia.org/wiki/GLib), but it's something quite different.)

Reply:
That's a typo. It should be GLIBC

Reply:
Fixed in https://github.com/python/cpython/pull/92618